packages/neon_framework/lib/src/pages/login.dart

```diff
@@ -92,7 +93,7 @@ class _LoginPageState extends State<LoginPage> {
                         hintText: 'https://...',
                         labelText: NeonLocalizations.of(context).loginUsingServerAddress,
                         suffixIcon: IconButton(
-                          icon: const Icon(Icons.arrow_forward),
+                          icon: Icon(Icons.adaptive.arrow_forward),
```

Comment:
Should we still move this into the AdaptiveIcons class so we always have the same syntax? That could also avoid using different icons for the same thing in different places.

Reply:
Sure we could add a redirecting method.